src/pybert/pybert.py

```diff
@@ -53,43 +44,38 @@
     Property,
     Range,
     String,
-    Trait,
+    cached_property,
 )
+from traits.etsconfig.api import ETSConfig
 from traitsui.message import message
 
-import skrf as rf
-
-from pyibisami import __version__ as PyAMI_VERSION
-from pyibisami.ami.parser import AMIParamConfigurator
-from pyibisami.ami.model import AMIModel
-from pyibisami.ibis.file import IBISModel
-
-from pybert import __version__ as VERSION
-from pybert import __date__ as DATE
+from chaco.api import ArrayPlotData, GridPlotContainer
 from pybert import __authors__ as AUTHORS
 from pybert import __copy__ as COPY
-
-from pybert.sim.simulation import my_run_simulation
-from pybert.visual.help import help_str
-from pybert.visual.plot import make_plots
+from pybert import __date__ as DATE
+from pybert import __version__ as VERSION
+from pybert.gui.help import help_str
+from pybert.gui.plot import make_plots
+from pybert.models.bert import my_run_simulation
 from pybert.utility import (
-    calc_G,
     calc_gamma,
     import_channel,
+    interp_s2p,
     lfsr_bits,
     make_ctle,
     pulse_center,
     safe_log10,
-    trim_impulse,
-    submodules,
     sdd_21,
-    H_2_s2p,
-    interp_time,
-    cap_mag,
-    interp_s2p,
-    renorm_s2p,
+    trim_impulse,
 )
 
+# ETSConfig.toolkit = 'qt4'  # Yields unacceptably small font sizes in plot axis labels.
+# ETSConfig.toolkit = 'qt4.celiagg'   # Yields unacceptably small font sizes in plot axis labels.
+# ETSConfig.toolkit = 'qt.qpainter'  # Was causing crash on Mac.
+# ETSConfig.toolkit = 'qt.image'     # Program runs, but very small fonts in plot titles and axis labels.
+# ETSConfig.toolkit = 'wx'           # Crashes on launch.
```

Comment:
These, when uncommented, _must_ execute before anything else.

Reply:
I initially missed this but now corrected.